docs/userguide/harness/user/gettingstarted.rst

```diff
@@ -135,12 +143,26 @@ The `configs.yaml` datafile lists all of the configurations files.
               config: <full path>
               sleep: 2
               invalid: ['some words']
+            3:
+              jinja2_config: routing.j2
+              jinja2_arguments:
+                lstrip_blocks: true
+                trim_blocks: true
+                bgp_data:
+                    bgp_as: 100
+                    neighbor_ips: [
+                        '1.1.1.1', '2.2.2.2'
+                    ]
+              configure_arguments:
+                bulk: True
```

Comment:
might be helpful to have timeout as well since timeout is also one of popular ones here.
```suggestion
                bulk: True
                timeout: 180
```